openbas-api/src/main/java/io/openbas/rest/scenario/ScenarioApi.java

```diff
@@ -176,15 +168,18 @@ public void importScenario(@RequestPart("file") @NotNull MultipartFile file) thr
 
   // -- SIMULATION --
 
-  // region scenarios
   @GetMapping(SCENARIO_URI + "/{scenarioId}/exercises")
   @PreAuthorize("isScenarioObserver(#scenarioId)")
   public Iterable<ExerciseSimple> scenarioExercises(@PathVariable @NotBlank final String scenarioId) {
     Scenario scenario = this.scenarioService.scenario(scenarioId);
     return scenario.getExercises().stream().map(ExerciseSimple::fromExercise).toList();
   }
 
-  // endregion
+  @GetMapping(SCENARIO_URI + "/external_reference/{externalReferenceId}")
```

Comment:
Prefix by /opencti and add versioning (v1)
Keep the old URL with deprecated method
Add swagger doc to explain which endpoint is supported